netbench/cdk/src/main/java/com/aws/ClientStack.java

```diff
@@ -0,0 +1,29 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.aws;
+
+import software.constructs.Construct;
+import software.amazon.awscdk.Stack;
+import software.amazon.awscdk.StackProps;
+import software.amazon.awscdk.services.s3.Bucket;
+import software.amazon.awscdk.services.ecs.Cluster;
+import software.amazon.awscdk.services.ec2.Vpc;
+import software.amazon.awscdk.services.ecs.*;
+import software.amazon.awscdk.services.autoscaling.AutoScalingGroup;
+import software.amazon.awscdk.services.ec2.InstanceType;
+import software.amazon.awscdk.services.ec2.GatewayVpcEndpoint;
+import software.amazon.awscdk.services.ec2.GatewayVpcEndpointOptions;
+import software.amazon.awscdk.services.ec2.GatewayVpcEndpointAwsService;
+
+
+public class ClientStack extends Stack {
+    public ClientStack(final Construct parent, final String id) {
+        this(parent, id, null, null, null);
+    }
+
+    public ClientStack(final Construct parent, final String id, Bucket metricsBucket,
```

Comment:
Why not subclass `StackProps` with a `ClientStackProps` class? Then you can easily pass in the extra params

Reply:
Makes sense, I will try that